flyway/sql/V1___initial_schema.sql

```diff
@@ -1,6 +1,7 @@
 CREATE TABLE names (
   id SERIAL PRIMARY KEY,
-  name TEXT NOT NULL
```

Comment:
@mwaliullah-omnia, please revert this change in V1. All the changes should be in the new version (e.g. in V2). Make them similarly to ```addressed``` table. If you do so, there'll be no need in ```repair``` in ```docker-compose.yml``` file.

Reply:
Thank you @omilaeva  for your feedback again. Sorry, I totally forgot that I also changed the names table in V1schemal.  I have now restored the original names table. Please check my latest commit here: petteri-te/github-exercise@9a0e71b.